play_motion:
  motions:
    pregrasp_1:
      joints: [torso_lift_joint, arm_right_1_joint,
      arm_right_2_joint, arm_right_3_joint, arm_right_4_joint, arm_right_5_joint,
      arm_right_6_joint, arm_right_7_joint]
      points:
      - positions: [0.34, -1.11, 0.69, 2.44, 1.92, -1.37, 0.95, 0.29]
        time_from_start: 0.0
      - positions: [0.34, -0.72, -0.09, 1.48, 1.71, -1.45, 0.81, 0.29]
        time_from_start: 2.0
      - positions: [0.34, 0.30, -0.09, 1.48, 1.71, -1.45, 0.81, 0.29]
        time_from_start: 3.5
      - positions: [0.34, 0.56, -0.25, 1.39, 1.32, -1.78, -0.03, 0.29]
        time_from_start: 6.5
      - positions: [0.34, 0.56, -0.51, 1.43, 0.98, -2.07, 0.45, 2.07]
        time_from_start: 10.0
      - positions: [0.34, 0.61, -0.51, 1.43, 0.98, -2.07, 0.45, 2.07]
        time_from_start: 12.0
      - positions: [0.34, 0.63, -0.51, 1.43, 0.98, -2.07, 0.45, 2.07]
        time_from_start: 17

    pick_final_pose:
      joints: [torso_lift_joint, arm_right_1_joint,
      arm_right_2_joint, arm_right_3_joint, arm_right_4_joint, arm_right_5_joint,
      arm_right_6_joint, arm_right_7_joint]
      points:  
      - positions: [0.34, 1.16, -0.64, 1.50, 0.00, -1.70, 0.00, 0.00]
        time_from_start: 10.0
    
    initial_trajectory:
      joints: [ torso_lift_joint, arm_right_1_joint,
      arm_right_2_joint, arm_right_3_joint, arm_right_4_joint, arm_right_5_joint,
      arm_right_6_joint, arm_right_7_joint ]
      points:  
      - positions: [ 0.34,-1.11, 0.04, 1.43, 1.50, -0.37, -0.08, 0.00]
        time_from_start: 10.0
    
    reach_goal:
      joints: [ torso_lift_joint, arm_right_1_joint,
      arm_right_2_joint, arm_right_3_joint, arm_right_4_joint, arm_right_5_joint,
      arm_right_6_joint, arm_right_7_joint ]
      points:  
      - positions: [ 1.43, -0.39, -2.97, 0.85, -2.07, 0.25, 1.70]
        time_from_start: 10.0
